packages/demo-cards/card-space-layout/edit.js

```diff
@@ -0,0 +1,27 @@
+import { setComponentTemplate } from '@ember/component';
+import { precompileTemplate } from '@ember/template-compilation';
+import templateOnlyComponent from '@ember/component/template-only';
+import BoxelCardContainer from '@cardstack/boxel/components/boxel/card-container';
+import './edit.css';
+
+export default setComponentTemplate(
+  precompileTemplate(
+    `<section class="card-space-layout-edit">
+      {{#each-in @fields as |name Field|}}
+        <BoxelCardContainer @displayBoundaries={{true}}>
+          <:header>
+            {{name}}
+          </:header>
+          <:default>
+            <Field />
+          </:default>
+        </BoxelCardContainer>
+      {{/each-in}}
+    </section>`,
```

Comment:
For the edit templates, all the Boxel Card Container components are on the isolated edit.js file. They could also be on each card's edit.js file instead. Is there a preference for one approach over the other?